Tests/Clappr_Tests/Classes/Plugin/Core/Drawer/BottomDrawerPluginTests.swift

```diff
@@ -0,0 +1,122 @@
+import Quick
+import Nimble
+
+@testable import Clappr
+
+class BottomDrawerPluginTests: QuickSpec {
+
+    override func spec() {
+        describe(".BottomDrawerPlugin") {
+            context("#init") {
+                var core: CoreStub!
+                var plugin: BottomDrawerPlugin!
+
+                beforeEach {
+                    core = CoreStub()
+                    plugin = BottomDrawerPlugin(context: core)
+                }
+
+                it("has bottom as position") {
+                    expect(plugin.position).to(equal(.bottom))
+                }
+
+                it("has size with the same width and the half of the height from parentView") {
+                    core.view = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 100))
+
+                    expect(plugin.size).to(equal(CGSize(width: 320, height: 50)))
+                }
+            }
+
+            describe("#render") {
+                var core: CoreStub!
+                var plugin: BottomDrawerPlugin!
+
+                beforeEach {
+                    core = CoreStub()
+                    plugin = BottomDrawerPlugin(context: core)
+                    core.view = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 100))
+                }
+
+                it("has origin x on zero and y equal to parentView height") {
+                    plugin.render()
+
+                    expect(plugin.view.frame.origin).to(equal(CGPoint(x: .zero, y: 100)))
+                }
+
+                it("has a view frame size equals to his size") {
+                    plugin.render()
+
+                    expect(plugin.view.frame.size).to(equal(CGSize(width: 320, height: 50)))
+                }
+
+                context("when plugin request a height greater then the limit") {
+                    it("uses the height limit insted") {
```

Comment:
There are two typos in this test description:
In the context description, it should be ```context("when plugin requests a height greater than the limit") ```
```suggestion
                    it("uses the height limit instead") {
```